src/main/game/Player/Computer.java

```diff
@@ -6,14 +6,15 @@
 
 public class Computer {
 
-    public static Moves playMove() {
-        return Moves.getRandomMove();
-    }
-
-    public static String playMoveTwo() {
+    public static String computerMove() {
         String[] moves = {"rock", "paper", "scissors"};
         Random random = new Random();
         int randomMove = random.nextInt(moves.length);
         return moves[randomMove];
     }
+
+    public static Moves playMove() {
+        String move = Computer.computerMove();
```

Comment:
Is there a particular reason why `computerMove()` needs to return a String? 
From what I see in the implementation above, that should writeable with `Moves.values()` and return a `Moves` instance directly.